sovtokenfees/sovtokenfees/main.py

```diff
@@ -97,4 +97,26 @@ def filter_fees(ledger_id: int, txn: Any):
     node.master_replica.register_hook(ReplicaHooks.APPLY_PPR,
                                       three_pc_handler.check_recvd_pre_prepare)
 
+    # Pluggable request handlers initialization
```

Comment:
As for tokens, i think it would be better to organize all register's method in some of Bootstrap method/class.

Reply:
ok